### DISCUSSION TITLE
How to use vdb_view on windows?

### DISCUSSION THREAD

**Discussion by ResiduaLiu** [AcademySoftwareFoundation/openvdb#1607] [Q&A]
After running cmake with  -DOPENVDB_BUILD_VDB_RENDER=ON, I got a foler Openvdb in C/Progam Files
![image](https://user-images.githubusercontent.com/50384819/233824250-ea8dd2c1-df0a-45fb-8e0c-43871958e91e.png)
Then I put some vdb files in the bin folder:
![image](https://user-images.githubusercontent.com/50384819/233824252-e1ab7948-4ffe-46bc-8a7f-3466b66a2235.png)
Then I run
```
 ./vdb_view bunny.vdb
```
And nothing shows up, if I directly run the exe file, it says lacking of various dll file like tbb12.dll glew.dll and so on. I tried to include all the vcpkg path in the system variable path. Those errors go away but still nothing shows up.

**Comment by somdats** [comment]
@ResiduaLiu 
Well if you have compiled openvdb successfully, enabling the option openvdb_view option while compiling, it should definitely work. 
I got it working on a Windows PC,
run the cmd, navigate to your vdb_view exe, and then as usual vdb_view pathTo/vdb
I believe you must have installed the glew & glfw library, otherwise, openvdb_view should not have worked

**Reply by ResiduaLiu** [reply]
Thank you!! Now Iam using vdb view very well!!
